tests/utils/staking.go

```diff
@@ -131,10 +131,11 @@ func DeployAndInitializeERC20TokenStakingManager(
 				SubnetID:           subnet.SubnetID,
 				MaximumHourlyChurn: 0,
 			},
-			MinimumStakeAmount:   big.NewInt(0).SetUint64(1e6),
-			MaximumStakeAmount:   big.NewInt(0).SetUint64(10e6),
-			MinimumStakeDuration: uint64(24 * time.Hour),
-			RewardCalculator:     common.Address{},
+			MinimumStakeAmount:       big.NewInt(0).SetUint64(1e6),
+			MaximumStakeAmount:       big.NewInt(0).SetUint64(10e6),
+			MinimumStakeDuration:     uint64(24 * time.Hour),
+			MinimumDelegationFeeRate: big.NewInt(0).SetUint64(100),
```

Comment:
```suggestion
			MinimumStakeAmount:       big.NewInt(1e6),
			MaximumStakeAmount:       big.NewInt(10e6),
			MinimumStakeDuration:     uint64(24 * time.Hour),
			MinimumDelegationFeeRate: big.NewInt(100),
```

Reply:
In general I think using `big.NewInt(0).SetUint64()` is a safer practice than doing `big.NewInt()` directly since the latter relies on implicit conversion between types and particularly since scientific notation numbers are float64's and susceptible to precision loss issues.

That being said I agree that we should be consistent and I will update my additions to use the former pattern

Reply:
I agree that the only difference in practice here is  precision difference between int64 overflow and max uint64 which is not relevant. I will make the change